### ISSUE TITLE
Ability to use `inherit` to import contents but not overwrite

### ISSUE THREAD

**Issue by alisonmosky** [r-lib/roxygen2#1584]
I would like to be able to essentially import the details from another function into the current function I'm documenting. 

What i'm hoping for is something like 

`@details
This function is a wrapper function for function_A. After retrieving the data from function_A, this function further wrangles the data....
Details for the logic flow of function_A are shown below: 

`@inherit function_A details

This does not work. I either have to copy paste the details section from the other function or ONLY inherit the details from that function. 

Is there any way around this? Maybe `@inherit function_A details as @newsection` so that you can inherit the contents but rename it?

**Comment by yurovska** [comment]
I need something similar too.

What I want is to `@inherit other_function examples` and add a few more lines of code there.

The following does not work:

```
#' @inherit other_function examples
#'
#' # Adding additional code at the end of examples inherited above
#' my_function(
#'   arg1 = "bla bla bla",
#'   arg2 = FALSE
#' )
```

**Comment by sckott** [comment]
Same use case as @yurovska for me, would love to be able to inherit examples from another function docs and add more code as examples

**Comment by hadley** [comment]
Yeah, this is how all the `@import*` tags work. I agree that roxygen2 needs more tools for reusing just part of a documentation block, but I'm becoming increasingly convinced that adding more tags isn't the way.
